### DISCUSSION TITLE
Trays Icons, Native Flathub, and System Monitor

### DISCUSSION THREAD

**Discussion by valex199132** [elementary/triage#462] [Ideas]
Hello. First of all, I would like to congratulate everyone on the beautiful interface of Elementary OS. I started using Linux-based systems a few months ago and this was by far the most beautiful and cohesive system I've observed. As version number 8 is being developed, I would like to suggest three things for it: that the system should support icons in the tray for software that runs in the background; native support for Flathub enabled by default in the app store, as it has few apps; and a native software to monitor the system's processes, because this is very much needed when a software crashes and we have to forcibly terminate a process.

**Comment by mattbrambilla** [comment]
I agree on everything, but we need to analyze these integrations thoroughly.

1. Native System Monitor: It's essential for eOS, but it must adhere the design guidelines used for others native apps.
2. Integration with FlatHub: ok but we need to differentiate between native eOS apps (written in Vala) and the other from FlatHub.
3. Tray Icons: Absolutely agree.

**Comment by 1280px** [comment]
1. See here: https://github.com/orgs/elementary/discussions/248 — though, it suggests including a third-party app from a sole developer, which also contains protestware elements(!). Apart from that, eOS already features a CLI system monitor called `htop`.
2. I believe it is included to the OS8 roadmap already? At the very least, it is now featured in onbording app.
3. They were intentionally removed in OS 5 (iirc): https://github.com/orgs/elementary/discussions/273

**Comment by wmprado** [comment]
Not only that, they could also put something like applets on the panel, I used that a lot, especially for pomodoro techniques. When will application indicators or applets be added to the Elementary OS panel?

The rest is too perfect, that's all that's missing for me. I love Elementary and it is greatly increasing my productivity in every way, because of that I will become a donor and help here.

**Comment by danirabbit** [comment]
I’m going to go ahead and close this as completed. We ship Flathub by default as of OS 8, and we support background apps and have a system monitor as of 8.1

in the future please submit separate posts for each separate idea. It makes it difficult to track the progress of grouped posts like this and it’s confusing for folks to vote on it :)
